app/services/recurring_contribution/subscriptions/processor.rb

```diff
@@ -54,6 +54,7 @@ def paid_with_credit_card?
 
   def assign_attributes(with_save, attributes)
     subscription.assign_attributes(attributes)
+    subscription.user.cpf = subscription.donator_cpf
 
     subscription.save if with_save && !has_errors?
```

Comment:
totally, thanks again 😃. About this suggestion, @fladson, I can't do it because this method checks some other errors, e.g. the creation of a credit card on the pagarme's API. So, if I change this, that validation will be ignored. If you check the complete file you will have a better understanding about what I said. But if you can see other way to do this, please, tell me.